test/e2e/framework/network_pods.go

```diff
@@ -278,7 +278,7 @@ func (np *NetworkPod) buildTCPCheckListenerPod() {
 						"for i in $(seq 1 $BUFS_NUM);" +
 							" do echo [dataplane] listener says $SEND_STRING;" +
 							" done" +
-							" | nc -w $CONN_TIMEOUT -l -v -p $LISTEN_PORT -s 0.0.0.0 >/dev/termination-log 2>&1",
+							" | nc -w ${CONN_TIMEOUT}s -l -v 0.0.0.0 $LISTEN_PORT >/dev/termination-log 2>&1",
```

Comment:
I want to see if it fails or not. If it works, I'd revert to `nc` syntax so it works with older images, as you pointed out earlier. It will also be a good way to see if script works or not.